docs/diagrams/ViewStudentSequenceDiagram.puml

```diff
@@ -0,0 +1,61 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant "logic:LogicManager" as LogicManager LOGIC_COLOR
+participant "tutorAidParser:TutorAidParser" as TutorAidParser LOGIC_COLOR
+participant ":ViewStudentCommand" as ViewStudentCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant ":ModelManager" as ModelManager MODEL_COLOR
+end box
+
+box Ui UI_COLOR_T1
+participant "ui:UIManager" as UIManager UI_COLOR
+end box
+
+[-> LogicManager: execute(view -s 1)
```

Comment:
Should `view -s 1` be a String instead? This applies to the `parseCommand` method too

Reply:
Yes I have updated it thank you.